Source/Reporters/DataTableReporter.jsl

```diff
@@ -0,0 +1,139 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+
+Names Default To Here( 0 );
+
+/*
+	Class: UtDataTableReporter
+		---Prototype---
+		class UtDataTableReporter inherits UtReporter
+		---------------
+		Saves results from tests as rows in a JMP table
+*/
+Define Class(
+	"UtDataTableReporter",
+	Base Class( "UtReporter" ),
+	
+	//Table to store results
+	dt = .;
+	
+	_init_ = Method({}, 
+		this:dt = New Table( "Test Results",
+			Add Rows( 0 ),
+			New Script(
+				"Summary",
+				Tabulate(
+					Show Control Panel( 0 ),
+					Add Table( Row Table( Grouping Columns( :result ) ) )
+				)
+			),
+			New Column( "DateTime", Numeric, "Continuous", Format( "m/d/y h:m:s", 23, 0 ),
+				Input Format( "m/d/y h:m:s", 0 ), Set Values([]) ),
+			New Column( "label", Character, "Nominal", Set Values({}), Set Display Width(50) ),
+			New Column( "label separated", Expression, "None", Formula( Words(:label, "⮚" ) ), Set Display Width(75) ),
+			New Column("Label 1", Character, "Nominal", Formula(If(Is Missing(:Label 5) & N Items(:label separated[Row()]) >= 1, :label separated[Row()][1], N Items(:label separated[Row()]) >= 2, :label separated[Row()][1], .))),
+			New Column("Label 2", Character, "Nominal", Formula(If(Is Missing(:Label 5) & N Items(:label separated[Row()]) >= 2, :label separated[Row()][2], N Items(:label separated[Row()]) >= 3, :label separated[Row()][2], .))),
+			New Column("Label 3", Character, "Nominal", Formula(If(Is Missing(:Label 5) & N Items(:label separated[Row()]) >= 3, :label separated[Row()][3], N Items(:label separated[Row()]) >= 4, :label separated[Row()][3], .))),
+			New Column("Label 4", Character, "Nominal", Formula(If(Is Missing(:Label 5) & N Items(:label separated[Row()]) >= 4, :label separated[Row()][4], N Items(:label separated[Row()]) >= 5, :label separated[Row()][4], .))),
+			New Column("Label 5", Numeric, "Nominal", Format("Best", 9), Formula(If(N Items(:label separated[Row()]) < 3, ., Regex(:label separated[Row()][N Items(:label separated[Row()])], " *([0-9]+)", "1") == "1", Num(Regex(:label separated[Row()][N Items(:label separated[Row()])], " *([0-9]+)", "\1")), .))),
+			New Column( "test expr", Expression, "None", Set Values({}) ),
+			New Column( "description", Character, "Nominal", Set Values({}) ),
+			New Column( "lre", Expression, "None", Set Values({}) ),
+			New Column( "payload", Expression, "None", Set Values({}) ),
+			New Column( "result", Character, "Nominal", Set Values({}) )
+		);
```

Comment:
For performance reasons and to avoid current data table conflicts, I would make this table private. You can add a new method to call when you need a data view (either on button click or when the tests are complete).
```
New Data View = Method({},
	this:dt << New Data View;
);
```
For example:
```
ut global reporter = ut data table reporter();
Include("RunTests.jsl");
ut global reporter << New Data View;
```